easybuild/easyconfigs/e/ega-quickview/ega-quickview-20240620-GCC-12.3.0.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'ConfigureMake'
+
+name = 'ega-quickview'
```

Comment:
I would prefer using `EGA-QuickView` as software name (see also description)

fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/20888